terraform/deployments/logging/main.tf

```diff
@@ -0,0 +1,40 @@
+terraform {
+  cloud {
+    organization = "govuk"
+    workspaces {
+      tags = ["logging", "aws"]
+    }
+  }
+  required_version = "~> 1.10"
+  required_providers {
```

Comment:
We don't also need to add a provider for google, do we?

Reply:
The logging S3 bucket is mirrored to a GCP bucket, so we do need it